test/input_files/v4.1/passed/complexfile_passed_000.vcf

```diff
@@ -51,9 +51,9 @@
HETA=0.0093;ERATE=0.0013;LDAF=0.0156;AA=T;AN=2184;VT=SNP;RSQ=0.5220;AVGPOST=0.9811;SNPSOURCE=LOWCOV;AC=21;AF=0.01;ASN_AF=0.0035;AMR_AF=0.01;AFR_AF=0.01;EUR_AF=0.01	GT:DS:GL	0|0:0.000:-0.01,-1.49,-5.00	0|1:0.600:-0.26,-0.40,-1.30	0|0:0.000:-0.23,-0.41,-1.57	0|0:0.000:-0.01,-1.74,-5.00	0|0:0.050:-0.22,-0.42,-1.74	0|0:0.200:-0.48,-0.48,-0.48	0|0:0.000:-0.03,-1.21,-5.00	0|0:0.000:-0.22,-0.42,-1.77	0|0:0.000:-0.02,-1.46,-5.00	0|0:0.000:-0.01,-1.93,-5.00	0|0:0.050:-0.48,-0.48,-0.48	0|1:0.800:-2.14,-0.41,-0.22	0|0:0.000:-0.01,-1.66,-5.00	0|0:0.000:-0.01,-1.52,-5.00	0|1:0.900:-0.48,-0.48,-0.48	0|0:0.000:-0.00,-1.95,-5.00	0|0:0.000:-0.04,-1.10,-5.00	0|0:0.000:-0.01,-1.93,-5.00	0|0:0.000:-0.48,-0.48,-0.48	0|0:0.000:-0.02,-1.39,-5.00	0|0:0.000:-0.11,-0.66,-4.10	0|0:0.000:-0.48,-0.48,-0.48	0|0:0.000:-0.11,-0.67,-4.22	0|0:0.000:-0.02,-1.34,-5.00	0|0:0.000:-0.06,-0.88,-5.00	0|0:0.000:-0.02,-1.43,-5.00	0|0:0.000:-0.03,-1.22,-5.00	0|0:0.000:-0.02,-1.46,-5.00	0|1:0.700:-0.21,-0.43,-1.87	0|0:0.050:-0.21,-0.43,-1.84	0|0:0.050:-0.24,-0.41,-1.51	0|0:0.000:-0.06,-0.87,-5.00	0|0:0.000:-0.02,-1.40,-5.00	0|0:0.350:-0.48,-0.48,-0.48	0|0:0.000:-0.27,-0.45,-0.98	0|0:0.000:-0.48,-0.48,-0.48	0|0:0.000:-0.48,-0.48,-0.48	0|0:0.000:-0.19,-0.46,-2.03	0|0:0.000:-0.48,-0.48,-0.48	0|0:0.000:-0.48,-0.48,-0.48	0|0:0.000:-0.17,-0.49,-2.33	0|0:0.000:-0.03,-1.20,-5.00	0|0:0.000:-0.48,-0.48,-0.48	0|0:0.000:-0.00341828,-2.10568,-5	0|0:0.000:-0.00,-2.26,-5.00	0|0:0.050:-0.18,-0.48,-2.29	0|0:0.000:-0.00,-1.98,-5.00	0|0:0.000:-0.05,-0.94,-5.00	0|0:0.000:-0.06,-0.91,-5.00	0|0:0.000:-0.03,-1.13,-5.00	0|0:0.000:-0.187809,-0.493711,-1.51942	0|0:0.000:-0.05,-0.93,-5.00	0|0:0.000:-0.01,-1.60,-5.00	0|0:0.000:-0.03,-1.12,-5.00	0|0:0.000:-0.03,-1.12,-5.00	0|0:0.000:-0.00,-2.03,-5.00	0|0:0.000:-0.06,-0.86,-5.00	0|0:0.000:-0.10,-0.69,-4.10	0|0:0.000:-0.00,-2.41,-5.00	0|0:0.000:-0.18,-0.48,-2.47	0|0:0.000:-0.10,-0.68,-3.47	0|0:0.000:-0.05,-0.96,-5.00	0|0:0.000:-0.48,-0.48,-0.48	0|0:0.000:-0.02,-1.40,-5.00	0|0:0.000:-0.10,-0.67,-4.40	0|0:0.000:-0.48,-0.48,-0.48	0|0:0.000:-0.19,-0.46,-2.06	0|0:0.000:-0.11,-0.67,-4.10	0|0:0.000:-0.10,-0.69,-3.62	0|0:0.000:-0.10,-0.68,-4.10	0|0:0.000:-0.04,-1.11,-5.00	0|0:0.100:-0.48,-0.48,-0.48	0|0:0.000:-0.48,-0.48,-0.48	0|0:0.000:-0.10,-0.69,-4.70	0|0:0.050:-0.19,-0.45,-2.18	1|0:1.000:-5.00,-0.38,-0.24	0|0:0.000:-0.19,-0.46,-1.92	0|0:0.000:-0.11,-0.66,-4.10	0|0:0.050:-0.16,-0.52,-3.01	0|0:0.000:-0.06,-0.90,-5.00	0|1:0.750:-0.48,-0.48,-0.48	0|0:0.000:-0.01,-1.92,-5.00	0|0:0.000:-0.06,-0.87,-5.00	0|0:0.000:-0.00,-1.98,-5.00	0|0:0.000:-0.193562,-0.458895,-1.92082	0|0:0.000:-0.06,-0.91,-5.00	0|0:0.000:-0.00,-2.17,-5.00	0|0:0.000:-0.18,-0.47,-2.16	0|0:0.000:-0.02,-1.43,-5.00	0|0:0.000:-0.03,-1.18,-5.00	0|0:0.000:-0.48,-0.48,-0.48	0|0:0.000:-0.19,-0.46,-1.91	0|0:0.000:-0.48,-0.48,-0.48	0|0:0.050:-0.22,-0.42,-1.75	0|0:0.100:-0.48,-0.48,-0.48	0|0:0.000:-0.06,-0.90,-4.70	0|0:0.000:-0.06,-0.87,-5.00	0|0:0.000:-0.10,-0.68,-3.40	0|0:0.000:-0.20,-0.46,-1.71	0|0:0.000:-0.48,-0.48,-0.48
```

Comment:
I think it's worth having a multiallelic in this file. I guess you removed it because it failed on the GL field, given that it has 3 fields and it should have 6, but I would prefer adding those extra 3 fields. it doesn't matter that you add the same values doing an automatic replace